content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+### Using input in Script Canvas 
+
+You can create a Script Canvas graph that connects to Input Events. For more information about scripting with Script Canvas, refer to [Creating Gameplay with Script Canvas](/docs/user-guide/scripting/script-canvas).
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component** and add the **Script Canvas** component.
+
+1. In the Script Canvas component, for **Script Canvas Asset**, click the **Open in Script Canvas Editor** icon and create a new Script Canvas graph like the following.
+
+### Example Script Canvas graph
+
+In the following graph, the **Input Handler** node connects `Action` events to various **Transform** nodes.  The **Pressed**, **Held**, and **Released** outputs are triggered when the state of an Event Generator changes.
+
+![Example Script Canvas graph for the Input component](/images/user-guide/interactivity/input/sc-input-example.png)
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to Input Events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, click the **Open in Lua Editor** icon.
+
+1. In the Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
+
+### Example Lua script
+
+The following Lua script connects `Action` events to the individual functions `OnPressed`, `OnHeld`, and `OnReleased`.
+
+```lua
+local tutorial_input = {
+    Properties = {}
+}
+function tutorial_input:OnActivate()
+    self.ActionInputs = {}
+
+    self.ActionInputs.OnPressed = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 2.0)
+    end
+
+    self.ActionInputs.OnHeld = function(_, value)
+        TransformBus.Event.RotateAroundLocalZ(self.entityId, 0.01)
+    end
+
+    self.ActionInputs.OnReleased = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 1.0)
+    end
+
+    self.ActionInputHandler = InputEventNotificationBus.Connect(self.ActionInputs, InputEventNotificationId("Action"))
+end
+
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect to an Input Handler for the Input Event in your `.inputbindings` file, you can test player input in **O3DE Editor**.
+
+1. In the viewport or **Entity Outliner**, select your entity.
```

Comment:
_Same comments as for earlier procedures_
* What UI surface is this? Clarify.
* "select your entity"
  * Will the reader know what we mean here by "the entity"? Is this the `.inputbindings` file? Clarify.